docs/ucl_myriad.md

```diff
@@ -43,3 +49,13 @@ Then make sure that your bin PATH is executable, by placing the following line i
 ```bash
 export PATH=$PATH:/home/<YOUR_ID>/bin
 ```
+
+## Running an nf-core pipeline
+
+To run the pipeline, make sure to add `-profile ucl_myriad,singularity`. This will download and launch the [`ucl_myriad.config`](../conf/ucl_myriad.config) which has been pre-configured with a setup suitable for the myriad cluster, as well as use the correct container engine for myriad.
```

Comment:
Why not just set `singularity = true` in the config itself?

Reply:
Great, we will do that :)